cli/src/main/kotlin/Main.kt

```diff
@@ -141,4 +165,25 @@ object Main : CommandWithHelp() {
 
         println()
     }
+
+    private fun loadConfig(): OrtConfiguration {
+        val argConfig = ConfigFactory.parseMap(configArguments, "Command line").withOnlyPath("ort")
+        val systemConfig = ConfigFactory.systemProperties().withOnlyPath("ort")
+        val fileConfig = configFile?.expandTilde()?.let {
+            require(it.isFile) {
+                "The provided configuration file '$it' is not actually a file."
+            }
+
+            ConfigFactory.parseFile(it).withOnlyPath("ort")
+        }
+        val defaultConfig = ConfigFactory.parseResources("default.conf")
+
+        var combinedConfig = argConfig.withFallback(systemConfig)
+        if (fileConfig != null) {
```

Comment:
Maybe a one-liner? `fileConfig?.let { combinedConfig.withFallback(it) }`

Reply:
I don't think this makes the code easier to read.